pkg/domain/workflow.go

```diff
@@ -0,0 +1,24 @@
+package domain
+
+type WorkFlowResponseData struct {
+	Result       interface{} `json:"result"`
+	WorkflowId   string      `json:"workflowId"`
+	WorkflowName string      `json:"workflowName"`
+	WsClientId   string      `json:"wsClientId"`
+}
+
+type WorkflowResponse struct {
+	SpecVersion     string               `json:"specversion"`
+	Id              string               `json:"id"`
```

Comment:
nit: You have no linter to complain about "ID" vs "Id" : -). Resolve this if you don't care.

Reply:
Fixed